src/reducers/windowHandler.js

```diff
@@ -210,20 +210,34 @@ export default function windowHandler(state = initialState, action) {
                 }
             });
 
-        case types.UPDATE_DATA_PROPERTY:
+        case types.UPDATE_DATA_PROPERTY: {
+            let value;
+
+            if (typeof action.value === 'string') {
+               value = action.value;
+
+            } else if (action.property === 'standardActions') {
+                // TODO: Evaluate if standardActions of type Set
```

Comment:
Why not asking @PabloSichert  if we need this check? And if not, we can take it out.

Reply:
This check is needed as it is the fix for issue #1224 and the gist of this PR.

What I wanted to say is because this check is needed for `standardActions` (it now is a `Set` instead of an `Array` introduced in https://github.com/metasfresh/metasfresh-webui-frontend/commit/3601e43f423be5b15ac0fd6fe1a1a0791c4bcdbe), maybe we could change the type back to `Array` instead of adding this check. But then some code would have to be reverted, as the mentioned commit uses `Set`-only functions like `.has()`, so I just put my thought there as a comment instead of blowing up this PR

Reply:
The `Set` is a more suitable data structure for checking if it contains a certain element. However, since it‘s a rather small list to be checked, we can change it back to `Array` if it takes out complexity in other parts.

Reply:
hi guys, 

basically i've stumbled on "TODO" because IMHO is not a TODO. I mean, what shall we solve there in the future? For me this sounds like a TODO which will live there forever.

wdyt?

Reply:
You're right, a `TODO` comment should be a required issue in the code. I will use `HINT` for comments like that in the future. I tend to use a lot of `TODO`s in my code and most of the time it's a negligible issue like this one, so I will change that from now on :)

Reply:
I guess the point is not leaving a comment in the code, when you could ask that question in the PR right away and be done with it.